firestore/firestore_snippets/listen_changes.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+// [START fs_listen_changes]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/firestore"
+)
+
+// listenChanges listens to a query, returning the list of DocumentChange
+// in the first snapshot.
```

Comment:
What do you mean, the first snapshot? It's the snapshot with new changes, so it's not the first of something in my vision

Reply:
How do you suggest to rewrite this commit?

Reply:
If it's taken from Java then let's keep it

Reply:
In Java they have: Listen to a query, returning the list of DocumentChange **events** in the first snapshot.
They have event methods and we don't so I decided to delete **events**.